### ISSUE TITLE
Crash ArangoDB v3.1 instance with export and cursor API

### ISSUE THREAD

**Issue by tammoippen** [arangodb/arangodb#2211]
## My environment running ArangoDB
I'm using the latest ArangoDB of the respective release series:
- [ ] 2.8
- [X] 3.0
- [X] 3.1
- [ ] self-compiled devel branch

On this operating system:
- [X] Linux 
  - [X] docker - official docker library

### This is a http interface-related issue:
I'm using the http interface with this browser: `curl`    running on this OS: MacOS 10.10.5
- [X] authentication is enabled? Yes and No. Issue persists with both.
- [ ] using the cluster?
- [X] _system database? Also works with different database.
```sh
» curl --version
curl 7.43.0 (x86_64-apple-darwin14.0) libcurl/7.43.0 SecureTransport zlib/1.2.5
```

These are the steps to reproduce:
1) start arangodb: `docker run -p 8529:8529 --name arangodb -it -d -e ARANGO_NO_AUTH=1 arangodb:3.1 --log.level debug`
2) Create collection: `curl -X POST -d '{"name": "test"}' --dump - http://localhost:8529/_api/collection`
3) Add some data:
```sh
curl -X POST -d '{"hello": "world", "counter": 1}' --dump - http://localhost:8529/_api/document/test
curl -X POST -d '{"hello": "world", "counter": 2}' --dump - http://localhost:8529/_api/document/test
```
4) Export the collection with a `batchSize` smaller than the actual collection size, i.e. a curser has to be created: `curl -X POST -d '{"batchSize": 1}' --dump - http://localhost:8529/_api/export?collection=test`
5) Query the next element: `curl -X PUT --dump - http://localhost:8529/_api/cursor/<id>` (using the id from the previous command)

The following problem occurs:
In 5) I get as a response: `curl: (52) Empty reply from server` and the arangodb instance is shutdown. Using ArangoDB 3.0 has no such issues. I stumbled over this, when using the `collection.all` function from `python-arango` driver and first filed an [issue](https://github.com/joowani/python-arango/issues/29) there. But this seems to be an ArangoDB issue.

### Execution output (arango output at the end)
```sh
--- ~ » curl -X POST -d '{"name": "test"}' --dump - http://localhost:8529/_api/collection
HTTP/1.1 200 OK
Location: /_db/_system/_api/collection/test
Content-Type: application/json; charset=utf-8
Server: ArangoDB
Connection: Keep-Alive
Content-Length: 127

{"id":"130","name":"test","waitForSync":false,"isVolatile":false,"isSystem":false,"status":3,"type":2,"error":false,"code":200}%

--- ~ » curl -X POST -d '{"hello": "world", "counter": 1}' --dump - http://localhost:8529/_api/document/test
HTTP/1.1 202 Accepted
Location: /_db/_system/_api/document/test/154
Etag: "_ULfasHS---"
Server: ArangoDB
Connection: Keep-Alive
Content-Type: application/json; charset=utf-8
Content-Length: 52

{"_id":"test/154","_key":"154","_rev":"_ULfasHS---"}%

--- ~ » curl -X POST -d '{"hello": "world", "counter": 2}' --dump - http://localhost:8529/_api/document/test
HTTP/1.1 202 Accepted
Location: /_db/_system/_api/document/test/176
Etag: "_ULfa0FK---"
Server: ArangoDB
Connection: Keep-Alive
Content-Type: application/json; charset=utf-8
Content-Length: 52

{"_id":"test/176","_key":"176","_rev":"_ULfa0FK---"}%

--- ~ » curl -X POST -d '{"batchSize": 1}' --dump - http://localhost:8529/_api/export\?collection\=test
HTTP/1.1 201 Created
Server: ArangoDB
Connection: Keep-Alive
Content-Type: application/json; charset=utf-8
Content-Length: 144

{"error":false,"code":201,"result":[{"_id":"test/154","_key":"154","_rev":"_ULfasHS---","counter":1,"hello":"world"}],"hasMore":true,"id":"189"}%

--- ~ » curl -X PUT --dump - http://localhost:8529/_api/cursor/189
curl: (52) Empty reply from server
```

The output from ArangoDB:
```sh
--- ~ » docker run -p 8529:8529 --name arangodb -it -d -e ARANGO_NO_AUTH=1 arangodb:3.1 --log.level debug
88be08a838f1a52ac01172437859e03e8a21daf938329829575f6bbbc70d2e0d
--- ~ » docker logs -f arangodb
Initializing database...Hang on...
Database initialized...Starting System...
{config} checking '///etc/arangodb3/arangod.conf'
{config} checking override '///etc/arangodb3/arangod.conf.local'
{config} loading '///etc/arangodb3/arangod.conf'
2016-12-07T08:17:56Z [1] INFO ArangoDB 3.1.3 [linux] 64bit, using VPack 0.1.30, ICU 54.1, V8 5.0.71.39, OpenSSL 1.0.1t  3 May 2016
2016-12-07T08:17:56Z [1] INFO using SSL options: SSL_OP_CIPHER_SERVER_PREFERENCE, SSL_OP_TLS_ROLLBACK_BUG
2016-12-07T08:17:56Z [1] DEBUG file-descriptors (nofiles) hard limit is 1048576, soft limit is 1048576
2016-12-07T08:17:56Z [1] DEBUG permanently changing the gid to 999
2016-12-07T08:17:56Z [1] DEBUG permanently changing the uid to 999
2016-12-07T08:17:56Z [1] INFO file-descriptors (nofiles) hard limit is 1048576, soft limit is 1048576
2016-12-07T08:17:56Z [1] DEBUG using default language 'en_US'
2016-12-07T08:17:56Z [1] DEBUG {mmap} memory-mapped range 0x7f0451000000 - 0x7f0453000000 (33554432 bytes), file-descriptor 6
2016-12-07T08:17:56Z [1] DEBUG found footer, reached end of datafile '/var/lib/arangodb3/journals/logfile-3.db', current size 10992
2016-12-07T08:17:56Z [1] DEBUG {mmap} memory-mapped range 0x7f044f000000 - 0x7f0451000000 (33554432 bytes), file-descriptor 7
2016-12-07T08:17:56Z [1] DEBUG found footer, reached end of datafile '/var/lib/arangodb3/journals/logfile-4.db', current size 48
2016-12-07T08:17:56Z [1] DEBUG reading database parameters from file '/var/lib/arangodb3/databases/database-1/parameter.json'
2016-12-07T08:17:56Z [1] DEBUG database parameters: {"deleted":false,"id":"1","name":"_system"}
2016-12-07T08:17:56Z [1] DEBUG added document collection '_modules'
2016-12-07T08:17:56Z [1] DEBUG added document collection '_frontend'
2016-12-07T08:17:56Z [1] DEBUG added document collection '_aqlfunctions'
2016-12-07T08:17:56Z [1] DEBUG added document collection '_users'
2016-12-07T08:17:56Z [1] DEBUG added document collection '_jobs'
2016-12-07T08:17:56Z [1] DEBUG added document collection '_graphs'
2016-12-07T08:17:56Z [1] DEBUG added document collection '_statistics15'
2016-12-07T08:17:56Z [1] DEBUG added document collection '_routing'
2016-12-07T08:17:56Z [1] DEBUG added document collection '_statisticsRaw'
2016-12-07T08:17:56Z [1] DEBUG added document collection '_apps'
2016-12-07T08:17:56Z [1] DEBUG added document collection '_statistics'
2016-12-07T08:17:56Z [1] DEBUG added document collection '_queues'
2016-12-07T08:17:56Z [1] INFO JavaScript using startup '/usr/share/arangodb3/js', application '/var/lib/arangodb3-apps'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/collection'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/user'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/database/target-version'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/foxx/setup'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/foxx/teardown'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/foxx/install'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/foxx/uninstall'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/foxx/replace'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/foxx/upgrade'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/foxx/configure'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/foxx/configuration'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/foxx/set-dependencies'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/foxx/dependencies'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/foxx/development'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/foxx/tests'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/foxx/script'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/statistics/short'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/statistics/long'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/routing/reload'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/routing/routes'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/server/role'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/server/id'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/time'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/sleep'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_admin/echo'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_admin/long_echo'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/statistics'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/statistics-description'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/test'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/execute'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/tasks'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/configuration/notifications/versions'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/explain'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/transaction'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/collection'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/user'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/database/target-version'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/foxx/setup'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/foxx/teardown'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/foxx/install'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/foxx/uninstall'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/foxx/replace'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/foxx/upgrade'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/foxx/configure'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/foxx/configuration'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/foxx/set-dependencies'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/foxx/dependencies'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/foxx/development'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/foxx/tests'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/foxx/script'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/traversal'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/statistics/short'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/statistics/long'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/routing/reload'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/routing/routes'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/server/role'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/server/id'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/time'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/sleep'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_admin/cluster-test'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_admin/echo'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/clusterStatistics'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_admin/long_echo'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/statistics'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/statistics-description'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/test'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/cluster/health'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/history'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/cluster/getSecondary'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/cluster/replaceSecondary'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/cluster/swapPrimaryAndSecondary'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/cluster/numberOfServers'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/cluster/cleanOutServer'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/cluster/moveShard'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/cluster/shardDistribution'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/cluster/rebalanceShards'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/cluster/supervisionState'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/database'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/endpoint'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/index'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  ''
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/auth/reload'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/aql/reload'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/execute'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/simple/any'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/simple/near'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/simple/within'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/simple/within-rectangle'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/simple/fulltext'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/simple/by-example'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/simple/first-example'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/simple/first'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/simple/last'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/simple/range'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/simple/remove-by-example'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/simple/replace-by-example'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/simple/update-by-example'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/tasks'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/aqlfunction'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/configuration/notifications/versions'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/explain'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/transaction'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/traversal'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_admin/cluster-test'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/clusterStatistics'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/cluster/health'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/history'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/cluster/getSecondary'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/cluster/replaceSecondary'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/cluster/swapPrimaryAndSecondary'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/cluster/numberOfServers'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/cluster/cleanOutServer'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/cluster/moveShard'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/cluster/shardDistribution'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/cluster/rebalanceShards'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/cluster/supervisionState'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/database'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/endpoint'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/index'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  ''
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/auth/reload'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT  '_admin/aql/reload'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/simple/any'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/simple/near'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/simple/within'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/simple/within-rectangle'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/simple/fulltext'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/simple/by-example'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/simple/first-example'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/simple/first'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/simple/last'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/simple/range'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/simple/remove-by-example'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/simple/replace-by-example'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/simple/update-by-example'
2016-12-07T08:17:56Z [1] DEBUG created JAVASCRIPT prefix  '_api/aqlfunction'
2016-12-07T08:17:56Z [1] DEBUG running database init/upgrade
2016-12-07T08:17:56Z [1] INFO In database '_system': Found 14 defined task(s), 1 task(s) to run
2016-12-07T08:17:56Z [1] INFO In database '_system': state standalone/existing, tasks updateUserModels
2016-12-07T08:17:56Z [1] DEBUG {mmap} memory-mapped range 0x7f0446bf0000 - 0x7f0446ff0000 (4194304 bytes), file-descriptor 18
2016-12-07T08:17:56Z [1] DEBUG reached end of datafile '/var/lib/arangodb3/databases/database-1/collection-7-1373260109/journal-20.db' data, current size 328
2016-12-07T08:17:56Z [1] INFO In database '_system': existing cleanup successfully finished
2016-12-07T08:17:56Z [1] INFO using endpoint 'http+tcp://0.0.0.0:8529' for non-encrypted requests
2016-12-07T08:17:56Z [1] DEBUG bound to endpoint 'http+tcp://0.0.0.0:8529'
2016-12-07T08:17:56Z [1] DEBUG autoloading actions
2016-12-07T08:17:56Z [1] DEBUG {mmap} memory-mapped range 0x7f0445fef000 - 0x7f04463ef000 (4194304 bytes), file-descriptor 23
2016-12-07T08:17:56Z [1] DEBUG reached end of datafile '/var/lib/arangodb3/databases/database-1/collection-60-1708716664/journal-71.db' data, current size 2952
2016-12-07T08:17:56Z [1] DEBUG autoloading actions finished
2016-12-07T08:17:56Z [1] DEBUG {mmap} memory-mapped range 0x7f0443fef000 - 0x7f0445fef000 (33554432 bytes), file-descriptor 24
2016-12-07T08:17:56Z [1] DEBUG created datafile '/var/lib/arangodb3/journals/logfile-105.db' of size 33554432 and page-size 4096
2016-12-07T08:17:56Z [1] DEBUG autoloading actions
2016-12-07T08:17:56Z [1] DEBUG autoloading actions finished
2016-12-07T08:17:56Z [1] DEBUG {mmap} memory-mapped range 0x7f045f802000 - 0x7f045f902000 (1048576 bytes), file-descriptor 26
2016-12-07T08:17:56Z [1] DEBUG reached end of datafile '/var/lib/arangodb3/databases/database-1/collection-56-2388228624/journal-88.db' data, current size 136
2016-12-07T08:17:56Z [1] DEBUG {mmap} memory-mapped range 0x7f0441fef000 - 0x7f0443fef000 (33554432 bytes), file-descriptor 25
2016-12-07T08:17:56Z [1] DEBUG created datafile '/var/lib/arangodb3/journals/logfile-106.db' of size 33554432 and page-size 4096
2016-12-07T08:17:56Z [1] DEBUG connecting to ip endpoint 'http+ssl://www.arangodb.com:443'
2016-12-07T08:17:56Z [1] DEBUG {mmap} memory-mapped range 0x7f043ffef000 - 0x7f0441fef000 (33554432 bytes), file-descriptor 27
2016-12-07T08:17:56Z [1] DEBUG created datafile '/var/lib/arangodb3/journals/logfile-128.db' of size 33554432 and page-size 4096
2016-12-07T08:17:56Z [1] INFO ArangoDB (version 3.1.3 [linux]) is ready for business. Have fun!
2016-12-07T08:18:06Z [1] DEBUG Statistics: Installing regular tasks...
--- ~ »
```

**Comment by jsteemann** [comment]
I have reproduced the crash with 3.1. The problem is caused by the wrong API being called for the export cursor.
The first API call goes to POST `/_api/export`. This creates an export cursor.
The following API call goes to PUT `/_api/cursor`. This API is supposed to access a cursor created by `/_api/cursor`, not by `/_api/export`. This is also the reason for the crash. 
Calling PUT `/_api/export` instead of PUT `/_api/cursor` for the second call will work properly and not cause a crash.

I have however added a check on the server-side so there will be no crashes anymore when the wrong API is called. That check will become available in 3.1.4.

**Comment by tammoippen** [comment]
👍
